src/js/intro.js

```diff
@@ -152,6 +152,16 @@
             element.removeChild(element.lastChild);
         }
     }
+
+    /**
+     * detects strings, checks for both string and String instances
+     * this is unlike typeof(x) === 'string' which only accepts primitive strings
+     *
+     */
+    function isString(thing) {
+        return typeof thing === 'string' || thing instanceof String;
```

Comment:
@odinhb can you change this to `return Object.prototype.toString.call(thing) === '[object String]'`

Reply:
I realize I didn't mention it, but I changed to this implementation because the only advantage to the one I posted in the issue (that I could find) is that it works across frames/iframes. I couldn't imagine that mattering so I switched to this one.

Reply:
@odinhb Can you share a live sample showing this ?